source/sgp_mode/SGPDataNodes.h

```diff
@@ -32,6 +40,46 @@ emp::DataFile &SGPWorld::SetupTasksFile(const std::string &filename) {
   return file;
 }
 
+emp::DataFile &SGPWorld::SetupHostOutputFile(const std::string &filename) {
```

Comment:
This only includes output that is square right? So let's name it to reflect that, otherwise it seems like it captures all host output